providers/dns/nearlyfreespeech/internal/client_test.go

```diff
@@ -121,7 +131,12 @@ func TestClient_RemoveRecord(t *testing.T) {
 		"type": "TXT",
 	}
 
-	mux.Handle("/dns/example.com/removeRR", testHandler(params))
+	// The reason we're testing that we're getting this exact authHeader is
+	// so that we can detect if some change that is not expected to modify
+	// neither the request nor the authHeader does so. If you're changing
+	// the request, please ensure your change actually works against the
+	// real nfsn API before modifying this golden value.
```

Comment:
```suggestion
```

Reply:
Just to confirm: do you want me to remove these comments?

Reply:
yes

Reply:
There is a button on the UI to accept suggestions if you need it.